app/exchangers/isagri/isacompta/fixed_assets_exchanger.rb

```diff
@@ -178,6 +101,31 @@ def import
           w.check_point
         end
       end
+
+      private
+
+      # Trim account number following preferences
+      def normalize(number)
```

Comment:
Maybe also handle the case where the account number length is too short and extend it?
Then, it can be used in the initializer of the `Account` class.

Reply:
Maybe add a test?

Reply:
It's seems to be already done in the before_validation of `Account`, that's why i just handled the "too long" case